runner/android_junit_runner/java/androidx/test/internal/runner/TestRequestBuilder.java

```diff
@@ -505,6 +503,20 @@ public TestRequestBuilder(Instrumentation instr, Bundle bundle) {
     deviceBuild = Checks.checkNotNull(deviceBuildAccessor);
     this.instr = Checks.checkNotNull(instr);
     argsBundle = Checks.checkNotNull(bundle);
+
+    maybeAddLegacySuppressFilter();
+    // add legacy Suppress filer iff it is on classpath
+  }
+
+  private void maybeAddLegacySuppressFilter() {
+    try {
+      Class<? extends Annotation> legacySuppressClass =
+          (Class<? extends Annotation>)
+              Class.forName("android.test.suitebuilder.annotation.Suppress");
+      filter.intersect(new AnnotationExclusionFilter(legacySuppressClass));
```

Comment:
@brettchabot doesn't this return a `Filter` object which is ignored? i.e. missing `filter = filter.intersect(...)`